### DISCUSSION TITLE
Incorrect formatting despite same repo name

### DISCUSSION THREAD

**Discussion by StefanMPopp** [alshedivat/al-folio#2231] [Q&A] [answered]
The website is not formatted correctly and consists only of the landing page.

I changed only my name and the url and deleted the base-url: https://github.com/StefanMPopp/stefanmpopp/blob/master/_config.yml
I tried it with and without capitalization, to match the repo and account names perfectly, without any change.

![image](https://github.com/alshedivat/al-folio/assets/38962260/b0180ce8-4654-428a-bcf6-6244dbb486d4)

![image](https://github.com/alshedivat/al-folio/assets/38962260/a56074b9-8f59-43ce-8cc2-6039f9c7c9ca)

**Comment by george-gca** [comment, marked as answer]
Your repository name is wrong. Check [our documentation](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#installing-and-deploying).

**Reply by IsaacPincus** [reply]
I'm having the same issue, but I cannot for the life of me figure out how I've messed up the page naming. I created the site https://isaacpincus.github.io/ exactly as described in the documentation, as far as I can tell, and the repository is https://github.com/IsaacPincus/IsaacPincus.github.io. This seems to be in exactly the same format as Stefan's page above, but I'm still getting the same error. 

I'm really sorry in advance if this is a simple error I'm overlooking, but I've been staring at it for 20 minutes trying to figure it out.

**Reply by george-gca** [reply]
Check our [FAQ](https://github.com/alshedivat/al-folio/blob/main/FAQ.md#my-webpage-works-locally-but-after-deploying-it-is-not-displayed-correctly-css-and-js-are-not-loaded-properly-how-do-i-fix-that).

**Reply by IsaacPincus** [reply]
Thank you! I had not properly deleted the baseurl.
